Source/WebKit/Shared/WebCoreArgumentCoders.serialization.in

```diff
@@ -2899,8 +2899,16 @@ header: <WebCore/ScrollTypes.h>
     ReplacedByCustomScrollbar
 };
 
+#if !PLATFORM(COCOA)
+header: <WebCore/ScrollTypes.h>
+[Nested, CustomHeader] struct WebCore::ScrollbarColor {
+    WebCore::Color thumbColor;
+    WebCore::Color trackColor;
+};
+#endif
+
```

Comment:
Does it make sense to remove the equivalent in RemoteScrollingCoordinatorTransaction.serialization.in and just make this cross-platform?